### PR TITLE
3: Adds css file to repo

### PR DESCRIPTION
**Pull request by mark-carlson**
Adds a css file to the root/assets folder and adds a body selector with a specified font family.

### CODE REVIEW

File: assets/styles.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* This is our CSS file
+ */
+
+body {
```
Comment (MatthewKaus) [inline assets/styles.css:4]: **This looks great!**

Comment (kleranscoding) [reply assets/styles.css:4]: This looks good to me

**Comment by MatthewKaus** [comment]
This looks great!

**Review by kleranscoding — APPROVED** [review]

**Review by MatthewKaus — APPROVED** [review]
LGTM

**Pull request merged**